.github/scripts/check_genesis.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# Paths
+paths=( "./resources/genesis/bitcoin-regtest/evm.json" 
```

Comment:
```
>>> import glob
>>> lst = glob.glob('**/evm.json', recursive=True)
>>> print(lst)
['resources/genesis/mock/evm.json', 'resources/genesis/bitcoin-regtest/evm.json', 'resources/genesis/mock-dockerized/evm.json', 'resources/genesis/devnet/evm.json', 'resources/test-data/integration-tests-low-block-gas-limit/evm.json', 'resources/test-data/integration-tests/evm.json', 'resources/test-data/demo-tests/mock/evm.json', 'resources/test-data/demo-tests/bitcoin-regtest/evm.json', 'resources/test-data/integration-tests-low-max-l2-blocks-per-l1/evm.json', 'crates/sovereign-sdk/examples/test-data/genesis/integration-tests-low-limiting-number/evm.json', 'crates/sovereign-sdk/examples/test-data/genesis/integration-tests/evm.json', 'crates/sovereign-sdk/examples/test-data/genesis/demo-tests/mock/evm.json', 'crates/sovereign-sdk/examples/test-data/genesis/demo-tests/celestia/evm.json', 'crates/sovereign-sdk/examples/test-data/genesis/demo-tests/bitcoin/evm.json']
```

Reply:
I'm not sure if we should do this, as for example `resources/genesis/devnet/evm.json` folder is an exception as it is not affected by automated genesis construction. In this case, I think it would work as `make genesis` wouldn't attempt to change that file either, so I'm not sure.

Reply:
wdyt @eyusufatik

Reply:
yes. production genesis files e.g. devnet or mainnet

should be only produced by our trigger, not auto generated